### PR TITLE
Fix/seed data streamline

### PR DESCRIPTION
**Pull request by zyusaf**
Changed the seed data so it now has an audio, image and description. Also added in more auth tokens and a randomiser to help with api calls please review :)

### CODE REVIEW

File: server/db/seeds.js [hunk 1]
```diff
@@ -18,13 +18,23 @@ db.accounts.insertMany([
 db.topics.insertMany([
     {
         title: "animals",
-        word_list: ["tiger", "dog", "cat", "koala", "chicken", "turtle", "whale", "owl", "rooster", "pigeon", "zebra", "fox"],
+        word_list: ["shark", "dog", "cat", "koala", "otter", "turtle", "whale", "owl", "wolf", "walrus", "fish", "fox"],
```
Comment (JaSmith13) [inline server/db/seeds.js]: Can you pull 2 names out of this so there are only 10, just so the progress bar works

**Review by JaSmith13 — COMMENTED** [review]
might need to slack the auth randomiser for review, i take it you made it in auth.js so its not been pushed as its in the .gitignore

File: server/db/seeds.js (hunk 1 above)
Comment (zyusaf) [reply server/db/seeds.js]: sorry Ali forgot to but will do that right now

Comment (JaSmith13) [reply server/db/seeds.js]: no worries, i dont know if theres a way for me to do it from github but i couldnt see one

**Comment by zyusaf** [comment]
> might need to slack the auth randomiser for review, i take it you made it in auth.js so its not been pushed as its in the .gitignore

**Review by JaSmith13 — APPROVED** [review]

**Pull request closed without merging**